skyvern/forge/sdk/services/credentials.py

```diff
@@ -0,0 +1,189 @@
+import logging
+import os
+from typing import Optional
+
+from onepassword.client import Client
+
+LOG = logging.getLogger(__name__)
+
+
+async def resolve_secret(reference: str) -> str:
+    """
+    Resolve a 1Password secret reference.
+
+    Args:
+        reference: A 1Password reference in the format op://vault_id/item_id/field
+                  or a custom format vault_id:item_id
+
+    Returns:
+        The resolved secret value
+    """
+    token = os.getenv("OP_SERVICE_ACCOUNT_TOKEN")
```

Comment:
add `OP_SERVICE_ACCOUNT_TOKEN` to skyvern/config.py and do `settings.OP_SERVICE_ACCOUNT_TOKEN` here